app/models/questionable/question.rb

```diff
@@ -20,24 +20,25 @@ def #{type}?
     end
 
     def answers_for_user(user)
-      answers = self.answers.where(user_id: user.id)
-      answers.any? ? answers : [self.answers.build(user_id: user.id)]
+      answers_for_user = answers.where(user_id: user.id)
```

Comment:
Technically there's nothing wrong with this [a local variable and a method (containing such variable) can share the same name] but I wonder if it would be better to pick another name for this one to avoid any confusion.